observability/otel/otel-configuring-metrics-for-monitoring-stack.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-configuring-metrics-for-monitoring-stack"]
+= Configuring metrics for the monitoring stack
+include::_attributes/common-attributes.adoc[]
+:context: otel-configuring-metrics-for-monitoring-stack
+
+toc::[]
+
+As a cluster administrator, you can configure the OpenTelemetry Collector custom resource (CR) to perform the following tasks:
```

Comment:
Can only cluster admins do these tasks? If so, and if you change the included modules to procedures, you might want to add prereqs to each procedure indicating that you must have `cluster-admin` permissions.

Reply:
Thank you. I'm taking a note of this. I already have this prerequisite in some modules, so I can reuse it then.